prow/metrics/metrics.go

```diff
@@ -34,10 +34,11 @@ import (
 const metricsPort = 9090
 
 // ExposeMetrics chooses whether to serve or push metrics for the service
-func ExposeMetrics(component string, endpoint string, interval time.Duration) {
+func ExposeMetrics(component string, endpoint string, interval time.Duration, serveMetricsFlag bool) {
 	if endpoint != "" {
 		go pushMetrics(component, endpoint, interval)
-	} else {
+	}
+	if serveMetricsFlag {
```

Comment:
I think we want:

 - if a pushgateway endpoint is given, push the metrics
 - if a pushgateway endpoint is given and serveMetrics is true, also serve metrics
 - if a pushgateway endpoint is not given, serve metrics

Reply:
Will do in the afternoon